tests/test_asset_bindings.py

```diff
@@ -1,4 +1,13 @@
 from Asset import *
+import os
+import sys
+
+if sys.platform == 'darwin':
```

Comment:
Annoyingly, this is replicated across all the test scripts. It would be better to have the bulk of this logic just once... either as a tests.get_output_directory() function, or as a tests.set_library_path(config) function.